### PR TITLE
[stable/postgressql] Support database outside cluster via service

### PR DESCRIPTION
**Pull request by sirrapa**
… via a Service in the cluster

*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

**Comment by aparris30** [comment]
/assign @lachie83 

@lachie83: Is there someone else who can review this PR?

**Comment by aparris30** [comment]
@lachie83 who can merge this PR?

**Comment by lachie83** [comment]
@sirrapa I was hoping to hear back from the chart maintainers about this PR. Let me do a review

File: stable/postgresql/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
```
Comment (lachie83) [inline stable/postgresql/OWNERS]: How did you arrive at this list and should these names also be added to the maintainer list in the Chart.yaml?

File: stable/postgresql/README.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ $ helm install stable/postgresql
 
 ## Introduction
 
-This chart bootstraps a [PostgreSQL](https://github.com/docker-library/postgres) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+This chart either:
```
Comment (lachie83) [inline stable/postgresql/README.md]: This chart can function in two different modes.

**Comment by lachie83** [comment]
I'm concerned about the value of this change if it doesn't deploy an application. cc @kubernetes/charts-maintainers

**Review by lachie83 — CHANGES_REQUESTED** [review]

File: stable/postgresql/OWNERS (hunk 1 above)
Comment (aparris30) [reply stable/postgresql/OWNERS]: This was a test, I'll remove this file

**Comment by prydonius** [comment]
> I'm concerned about the value of this change if it doesn't deploy an application. cc 

I would agree with @lachie83, I would suggest creating a separate chart in incubator (`postgresql-proxy-service` or similar) to just provide service discovery for an external postgres server.

**Pull request closed without merging**